src/boot/efi/boot.c

```diff
@@ -1057,18 +1057,30 @@ static bool menu_run(
                 efivar_set(MAKE_GUID_PTR(LOADER), u"LoaderEntryDefault", config->entry_default_efivar, EFI_VARIABLE_NON_VOLATILE);
 
         if (console_mode_efivar_saved != config->console_mode_efivar) {
-                if (config->console_mode_efivar == CONSOLE_MODE_KEEP)
-                        efivar_set(MAKE_GUID_PTR(LOADER), u"LoaderConfigConsoleMode", NULL, EFI_VARIABLE_NON_VOLATILE);
-                else
+                if (config->console_mode_efivar == CONSOLE_MODE_KEEP) {
+                        _cleanup_free_ char *buf = NULL;
```

Comment:
given you are not using the return values, just pass NULL for both, you'll still get the success/failure